Zend/zend_attributes.c

```diff
@@ -91,6 +94,59 @@ ZEND_METHOD(AllowDynamicProperties, __construct)
 	ZEND_PARSE_PARAMETERS_NONE();
 }
 
+ZEND_METHOD(SensitiveParameter, __construct)
+{
+	ZEND_PARSE_PARAMETERS_NONE();
+}
+
+ZEND_METHOD(SensitiveParameterValue, __construct)
+{
+	zval *value;
+
+	ZEND_PARSE_PARAMETERS_START(1, 1)
+		Z_PARAM_ZVAL(value)
+	ZEND_PARSE_PARAMETERS_END();
+
+	ZVAL_COPY(OBJ_PROP_NUM(Z_OBJ_P(ZEND_THIS), 0), value);
+}
+
+/* {{{ */
+ZEND_METHOD(SensitiveParameterValue, getValue)
+{
+	ZEND_PARSE_PARAMETERS_NONE();
+
+	ZVAL_COPY(return_value, OBJ_PROP_NUM(Z_OBJ_P(ZEND_THIS), 0));
+} /* }}} */
+
+/* {{{ */
+ZEND_METHOD(SensitiveParameterValue, __debugInfo)
+{
+	ZEND_PARSE_PARAMETERS_NONE();
+
+	array_init(return_value);
```

Comment:
```suggestion
	RETURN_EMPTY_ARRAY();
```

Your code works, but this is slightly more efficient and returns an immutable empty array instead of allocating a brand new array.

Same for the other methods that don't populate an array later.